serving/samples/helloworld-rust-nightly/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# ---------- Build Container ----------
```

Comment:
The goal of Hello World samples are to give simple examples. I am not sure we need a multi-stage Dockerfile for this use case.

Reply:
It is very simple, and also a best practice, and really more examples should be showing how to run code on scratch containers to keep resources and costs under control.